tutorials/math/mathcoreVectorFloatIO.C

```diff
@@ -117,7 +117,7 @@ void runIt() {
 }
 
 void mathcoreVectorFloatIO() {
-#if defined(__CINT__) && !defined(__MAKECINT__)
+#if defined(__ICLING__) && !defined(__MAKECLING__)
```

Comment:
```suggestion
#if defined(__CLING__) && !defined(__ROOTCLING__)
```

Reply:
For now let's split the PR between documentation changes from the functional changes (where things seems to be a bit more messy and need some clarification).